code/scripting/api/libs/ui.cpp

```diff
@@ -2805,5 +2807,110 @@ ADE_FUNC(getHelpText, l_UserInterface_MultiPXO, nullptr, "Gets the help text lin
 	return ade_set_args(L, "t", pages);
 }
 
+//**********SUBLIBRARY: UserInterface/MultiStartGame
+ADE_LIB_DERIV(l_UserInterface_MultiStartGame,
+	"MultiStartGame",
+	nullptr,
+	"API for accessing data related to the Multi Start Game UI.",
+	l_UserInterface);
+
+ADE_FUNC(initMultiStart, l_UserInterface_MultiStartGame, nullptr, "Initializes the Create Game methods and variables", nullptr, nullptr)
+{
+	SCP_UNUSED(L);
+
+	multi_start_game_init(true);
+
+	return ADE_RETURN_NIL;
+}
+
+ADE_FUNC(closeMultiStart, l_UserInterface_MultiStartGame, "boolean Start_or_Quit", "Finalizes the new game settings and moves to the host game UI if true or cancels if false. Defaults to true.", nullptr, nullptr)
+{
+	bool choice = true;
+	if (!ade_get_args(L, "b", &choice))
+		return ADE_RETURN_NIL;
+
+	if (choice) {
+		multi_start_game_close(true);
+	} else {
+		multi_quit_game(PROMPT_NONE);
+	}
+
+	return ADE_RETURN_NIL;
+}
+
+ADE_FUNC(runNetwork, l_UserInterface_MultiStartGame, nullptr, "Runs the network required commands to update the status text", nullptr, nullptr)
+{
+	SCP_UNUSED(L);
+
+	multi_start_game_do(true);
+
+	return ADE_RETURN_NIL;
+}
+
+ADE_FUNC(setName, l_UserInterface_MultiStartGame, "string Password", "Sets the game's name", "boolean", "True if successful, false otherwise")
```

Comment:
"Password"? Guessing it should be "Name" or similar.

Reply:
Good catch. Fixed.